shared/devices/add-device/index.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import {isLargeScreen} from '../../constants/platform'
+
+type Props = {|
+  onAddComputer: () => void,
+  onAddPaperKey: () => void,
+  onAddPhone: () => void,
+  onCancel: () => void,
+  title: string,
+|}
+
+const AddDevice = (props: Props) => (
+  <Kb.ScrollView alwaysBounceVertical={false}>
+    <Kb.Box2
+      direction="vertical"
+      gap="medium"
+      alignItems="center"
+      style={styles.container}
+      gapStart={true}
+      gapEnd={true}
+    >
+      <Kb.Box2 direction="vertical" gap="tiny" alignItems="center">
+        {!Styles.isMobile && <Kb.Text type="Header">Add a device</Kb.Text>}
+        <Kb.Text type="Body" center={true}>
+          Protect your account by having more devices and paper keys.
+        </Kb.Text>
+      </Kb.Box2>
+      <Kb.Box2
+        direction={Styles.isMobile ? 'vertical' : 'horizontal'}
+        gap="large"
+        gapStart={true}
+        gapEnd={true}
+      >
+        <DeviceOption onClick={props.onAddComputer} type="computer" />
+        <DeviceOption onClick={props.onAddPhone} type="phone" />
+        <DeviceOption onClick={props.onAddPaperKey} type="paper key" />
+      </Kb.Box2>
+    </Kb.Box2>
+  </Kb.ScrollView>
+)
+
+const DeviceBox = Styles.isMobile
+  ? Kb.Box2
+  : Styles.styled(Kb.Box2)({
+      ...Styles.transition('background-color'),
+      '&:hover': {
+        backgroundColor: Styles.globalColors.blue4,
```

Comment:
can just use css for this right, we generate the background hover css automatically

Reply:
I made the other changes in https://github.com/keybase/client/pull/15972/commits/4162e5de421fbe3afa86547860709fc564f1d85e, I can't find the bg hover css generation

Reply:
https://github.com/keybase/client/pull/15972/commits/388ca0ebeca57598a7433b714c2561735b40f95b uses className instead